specification/storagesync/resource-manager/Microsoft.StorageSync/stable/2020-03-01/storagesync.json

```diff
@@ -3071,6 +3071,66 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.StorageSync/locations/{locationName}/operations/{operationId}": {
```

Comment:
@anpint 
Your spec defines operations API as /subscriptions/{subscriptionId}/providers/Microsoft.StorageSync/locations/{locationName}/operations/{operationId}
But the examples use operationResults in the ID. Which one is correct?

Can you point me to your S360 error?

Reply:
As per our offline discussion I understand the resource name 'operations' is correct and is what your shipped API is using for Azure-AsyncOperation format. The resourceName should be 'operationStatuses'. Refer https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#azure-asyncoperation-resource-format

The resource name 'operations' confuses with RP general operations API. Please add a work item to fix the resource name in future API versions. 

I am signing off now as this is an existing API and you are trying to fix S360 error.